adapters/http-vertx-base/src/main/java/org/eclipse/hono/adapter/http/AbstractVertxBasedHttpProtocolAdapter.java

```diff
@@ -860,18 +879,21 @@ protected Integer getTimeUntilDisconnectFromRequest(final RoutingContext ctx) {
      * @param deviceId The identifier of the device.
      * @param currentSpan The <em>OpenTracing</em> Span used for tracking the processing of the request.
      */
-    private void addConnectionCloseHandler(
+    private void setTtdRequestConnectionCloseHandler(
```

Comment:
this handler seems to be useful/required for any request, not just TTD requests, doesn't it?